cmd/tc-redirect-tap/main_test.go

```diff
@@ -216,7 +216,7 @@ func TestGetCurrentResult(t *testing.T) {
 		StdinData: rawPrevResultBytes,
 	}
 
-	actualResult, err := getCurrentResult(cmdArgs)
+	actualResult, _, err := getCurrentResult(cmdArgs)
```

Comment:
We can validate here the the returned result matches the cni version in the conf.